docs/javascript/browser-apis-and-jquery.md

```diff
@@ -956,3 +956,122 @@ setTimeout(removeEvent, 5000);
 ![offMethod example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/offmethod.gif?raw=true){ width="100%" }
 <figcaption></figcaption>
 </figure>
+
+## Getting and Setting attributes
+
+### The `getAttribute()` function allows you to access the value of an element's attribute
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Home</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+  <div class="object" id="one"></div>
+  <img class="object" id="two">
+
+</div>
+<script>
+
+  var elementAttribute = document.querySelector('div').getAttribute("id");
+  console.log("Attribute:", elementAttribute);
+
+</script>
+</html>
+```
+
+Example Output:
+
+```javascript
+Attribute: one;
+```
+
```

Comment:
Can we also show the `src` attribute, since you're going to use it so much below?

Reply:
For this one, I was meaning something like:

```javascript
var imageElement = document.getElementById("one");
imageElement.src = "dog.png";
```

Reply:
I'll add it in in another PR

Reply:
Ah ok, I'll have to think of another example to use the setAttribute function.